File: Iniciante/URI 1963.r
input = file('stdin', 'r')

line = strsplit(readLines(input, n=1), " ")
a = as.double(line[[1]][1])
b = as.double(line[[1]][2])
aumento = (b * 100.0 / a) - 100.0;

write(sprintf("%.2f%%", aumento), '')

File: Iniciante/URI 1963.php
<?php
    $line = explode(" ", trim(fgets(STDIN)));
    $A = floatval($line[0]);
    $B = floatval($line[1]);
    $aumento = ($B * 100.0 / $A) - 100.0;
    $aumento = number_format($aumento, 2, ".", "");
    echo "$aumento%\n";
?>